meetings/2023-04-19.md

```diff
@@ -0,0 +1,131 @@
+* SC: Solid protocol just happens to refer to this work.
+* TT: The normatively referenced specifications should be no more than one step behind the specification under consideration.
+* SC: I've checked some documents about the transition and the process but couldn't see it clearly. Might be an unwritten rule. That's how I remember it as well but would be useful to have a pointer to this requirement or guideline.
+* PA: +1 to what Ted says. I am trying to find a normative reference for this rule. If the Solid protocol needs a normative reference, they need to be mature enough to be recommended by the Working Group. That is up to the WG to define. For other CG's it is more complicated.
+* SC: Currently the PR is worded such that these are tentative deliverables, which have to be considered by the WG.
+* EP: We have to be coherent in expectations on ourselves. If we follow this rule, that normative references can be at most one step behind, if no other WG or proposed WG is addressing this then the Solid WG will have to address this itself. We need to decide what we think are "hard dependencies" for the Solid protocol, i.e. what references are required for a conformant implementation.
+* SC: Agree with EP, the Solid Protocol will need to reference certain specifications. Guidelines on the status thereof needs to come from W3C, and the WG will have to conform to that. We shouldn't make this decision here. For example: If the WG decides Solid-OIDC or N3 is not desireable as a reference, then they can seek to fill this gap with other specifications. 
+* PA: +1 to the consistency argument. We must be consistent in the process. We have to mention these CG specifications that are currently normative references. What we should not commit to in the charter is whether those references have to become specifications of their own. The idea is that the separation in documents might not match the current separation of the CG specifications. The charter should give a high-level view of what the WG is expected to address. We should not be too specific in how these references are to be used in the final result.
+* SC: Would you say that those specifications that are listed in the purview of the Solid CG should be listed as a tentative deliverable in the charter, as they currently are in the PR?
+* EP: There might be a benefit in not being over-specific in how we are to deliver the scope. For example it might be beneficial to extract the storage definition, similar to WAC, ACP, ..., where the protocol groups these different specifications. Maybe we just don't have to commit yet at the start on the how, and focus on the scoping right now.
+* EP: Question to PA: Looking at the deliverables section in the [charter](http://htmlpreview.github.io/?https://github.com/solid/solid-wg-charter/blob/12a9e08ff5876e24c0f1b7c0f180d3e1bc5b6efa/charter/index.html) how specific should we be?
```

Comment:
```suggestion
* EP: There might be a benefit in not being over-specific in how we are to deliver the scope. For example it might be beneficial to extract the storage definition, similar to WAC, ACP, etc., where the protocol groups these different specifications. Maybe we just don't have to commit at the start on the how, and focus on the scoping right now.
* EP: Question to PA: Looking at the deliverables section in the [charter](http://htmlpreview.github.io/?https://github.com/solid/solid-wg-charter/blob/12a9e08ff5876e24c0f1b7c0f180d3e1bc5b6efa/charter/index.html), how specific should we be?
```